crates/iceberg/src/io/storage_gcs.rs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//! Google Cloud Storage properties
+
+use std::collections::HashMap;
+
+use opendal::services::GcsConfig;
+use opendal::Operator;
+
+use crate::{Error, ErrorKind, Result};
+
+// Reference: https://github.com/apache/iceberg/blob/main/gcp/src/main/java/org/apache/iceberg/gcp/GCPProperties.java
+
+/// Google Cloud Storage bucket name
+pub const GCS_BUCKET: &str = "gcs.bucket";
+/// Google Cloud Storage endpoint
+pub const GCS_ENDPOINT: &str = "gcs.endpoint";
```

Comment:
Shoud be:

```java
GCS_SERVICE_HOST = "gcs.service.host";
```

Reply:
Ah! Thank you, this is useful to know.

I was unsure whether to follow the [Python config](https://py.iceberg.apache.org/configuration/#google-cloud-storage) or Java. I'll go with Java from these comments :+1:

Reply:
cc @Fokko for double check.